frontend/lib/search/search_page.dart

```diff
@@ -51,17 +58,28 @@ class _SearchPageState extends State<SearchPage> {
             }
           });
         }),
-        ResultsLoader(
-            searchText: _searchFieldText,
-            categoryIds: _selectedCategories.map((e) => e.id).toList())
-      ],
+        (_locationStatus == LocationRequestStatus.RequestFinished)
+            ? ResultsLoader(
+                searchText: _searchFieldText,
+                categoryIds: _selectedCategories.map((e) => e.id).toList(),
+                coordinates: _coordinates)
+            : null
+      ].where((element) => element != null).toList(),
     );
   }
 
   @override
   void initState() {
     _textEditingController = TextEditingController();
     _focusNode = FocusNode();
+    determinePosition()
+        .then((value) => setState(() => {
+              _coordinates =
+                  CoordinatesInput(lat: value.latitude, lng: value.longitude)
+            }))
+        .whenComplete(() => this.setState(() {
```

Comment:
Should we display an error when the lookup of the location "failed". If there is no permission then this is fine. But there could also be errors afaik.

Reply:
Hmmm.. yeah, the current fallback is using alphabetic sorting of the results :thinking: Could definitely be polished